yt_dlp/extractor/youtube.py

```diff
@@ -2262,8 +2265,11 @@ def feed_entry(name):
             'categories': [category] if category else None,
             'tags': keywords,
             'is_live': is_live,
+            'is_upcoming': is_upcoming,
```

Comment:
I am not sure this is usefull. User can simply check if `timestamp` is in the future

Reply:
Fields like this one can be pretty handy in situations where the data is consumed by human eyes, offering helpful information at a glance.